src/App.test.js

```diff
@@ -1,8 +1,11 @@
-import { render, screen } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+// Testeamos el logo mediante la propiedad atributo alt
+
+describe('Ecommerce Store logo/name appears in the Header', () => {
+  it('Logo', () => {
+    const {getByAltText} = render(<App/>)
+    getByAltText("shoppingCarIcon");
```

Comment:
Here you can make an assertion on the queried element, and use jest matchers, a very common is `expect(whatGetByAltTextReturns).toBeInTheDocument()` so the unit test really ensures the componente is being rendered. Take a look at [this](https://testing-library.com/docs/ecosystem-jest-dom/)